politeiawww/cmd/politeiawww_dbutil/politeiawww_dbutil.go

```diff
@@ -593,6 +655,27 @@ func validateCockroachParams() error {
 	return nil
 }
 
+func validateMySQLParams() error {
+	// Validate host.
+	_, err := url.Parse(*mysqlhost)
+	if err != nil {
+		return fmt.Errorf("parse host '%v': %v", *mysqlhost, err)
+	}
+
+	// Validate password.
+	if *password == "" {
+		return fmt.Errorf("MySQL password missing; use -password flag to" +
+			" provide it")
+	}
+
+	// Ensure encryption key file exists.
+	if !util.FileExists(*encryptionKey) {
```

Comment:
Isn't the encryption key optional?  This doesn't handle the case of no encryption key being present.

Reply:
Looks like it's required - dbuutil/cockroachdb & politeiawww/cockroach also try load encryption key.